internal/controller/warehouses/warehouses.go

```diff
@@ -151,6 +155,7 @@ func newReconciler(
 	r.getLatestFreightFromReposFn = r.getLatestFreightFromRepos
 	r.selectCommitsFn = r.selectCommits
 	r.getLastCommitIDFn = r.getLastCommitID
+	r.getDiffPathsSinceCommitIDFn = r.getDiffPathsSinceCommitID
```

Comment:
Tests should probably assert that this is set by `newReconciler()`.

Reply:
I believe this is already done:

https://github.com/akuity/kargo/blob/407be79fb9dccaf77264043bfca37795cd3ca4dd/internal/controller/warehouses/warehouses_test.go#L43

Reply:
Don't know how I missed that! 👍